apps/bare-expo/android/app/src/main/java/dev/expo/payments/MainActivity.java

```diff
@@ -20,15 +20,11 @@ protected String getMainComponentName() {
 
   @Override
   protected ReactActivityDelegate createReactActivityDelegate() {
-    return new ReactActivityDelegate(this, getMainComponentName()) {
+    return DevelopmentClientController.getInstance().wrapReactActivityDelegate(this, new ReactActivityDelegate(this, getMainComponentName()) {
```

Comment:
Should this be conditioned on `USE_DEV_CLIENT` just in the 'bare-expo' app case?